admin/class-wpadcenter-admin.php

```diff
@@ -1686,4 +1686,91 @@ public function wpadcenter_ad_group_selected() {
 		}
 	}
 
+	/**
+	 * Ajax when ad is selected in reports custom-reports page.
+	 */
+	public function wpadcenter_ad_selected() {
+		// check nonce security.
+		if ( isset( $_POST['action'] ) ) {
+			check_admin_referer( 'selectad_security', 'security' );
+		}
+
+		if ( 'selected_ad_reports' === $_POST['action'] ) {
+			$start_date = isset( $_POST['start_date'] ) ? gmdate( 'Y-m-d', intval( $_POST['start_date'] ) ) : '';
```

Comment:
Add validations

Reply:
start_date value coming from front end is an integer ( timestamp ) and therefore intval() function is used as sanitization